rakelib/artifacts.rake

```diff
@@ -208,9 +208,11 @@ namespace "artifact" do
     Rake::Task["artifact:zip_oss"].invoke
     Rake::Task["artifact:tar"].invoke
     Rake::Task["artifact:tar_oss"].invoke
-    Rake::Task["artifact:docker"].invoke
-    Rake::Task["artifact:docker_oss"].invoke
-    Rake::Task["artifact:dockerfile"].invoke
+    unless ENV['SKIP_DOCKER'] == "1" do
```

Comment:
LGTM pending the removal of the `do` on this line and green CI builds.

Reply:
I clearly can't ruby.